website/docs/extensions/testing.md

```diff
@@ -19,15 +19,27 @@ The behaviour of the `scarb test` command can be changed by developers.
 To do so, provide a script named explicitly `test` in the current workspace `Scarb.toml`.
 If such script is found, Scarb will execute it instead of running the default test runner.
 
-For example, in order to tell `scarb test` to use [Starknet Forge](https://foundry-rs.github.io/starknet-foundry) for testing in
-your project, type the following:
+Scarb can be configured to use any tool in place of the default `cairo-test`, simply by changing
+this configuration:
 
 ```toml filename="Scarb.toml"
 [scripts]
-test = "snforge"
+test = "my-custom-command"
 ```
 
-Do not forget to properly set up Starknet Forge in your project beforehand.
+## Using Starknet Foundry
+
+[Starknet Foundry](https://foundry-rs.github.io/starknet-foundry) is another project developed by [Software Mansion](https://swmansion.com/) team.
+It enables advanced testing of [Starknet](https://www.starknet.io/) contracts, including fuzz testing, forking the network state and many more.
```

Comment:
```suggestion

It is a fast test runner designed to work well with Scarb. It enables advanced testing of [Starknet](https://www.starknet.io/) contracts, including cheatcodes, fuzz testing, forking the network state and tooling to deploy your contracts to the network and interact with them.
```

Reply:
I don't think we should assume that it's universally understood what `cheatcodes` mean. I think it should be written in a more descriptive way.

Reply:
Agreed, I would just remove it but leave the part about interactions

Reply:
I think generally we could also create a section about deploying the contracts? maybe in the Starknet extension chapter?

Reply:
I am suggesting it because I had a perception people were asking for a scarb deploy in the past, so that would save you some explanations. If it is not the case anymore I am indifferent to this proposal.

Reply:
> people were asking for a scarb deploy in the past

never heard of this, definitely not in recent months

but I think we might indeed welcome a page saying "You have compiled a contract, what to do next?" and it will have links to SNFoundry, Starkli etc. Wdyt?

Reply:
Yup I thought about something like this.

Reply:
topic for separate PR, @cptartur will you tackle this?

Reply:
I can do it in a separate PR as suggested.

Reply:
I've updated the Starknet Foundry description so it explains cheatcodes without calling them explicitly. I don't want to include anything about deploying contracts here because it has nothing to do with testing. It will be included in a different place in a separate PR.